examples/windowing/windowExample/src/ofApp.cpp

```diff
@@ -39,41 +42,39 @@ void ofApp::update(){
 		ofShowCursor();
 	}
 	
-	
 	ballPositionX += ballVelocityX;
 	ballPositionY += ballVelocityY;
 	
-	int posx = ofGetWindowPositionX();
-	int posy = ofGetWindowPositionY();
```

Comment:
does this point to a bug/regression with ofGetWindowPositionX / ofGetWindowPositionY ?

This code used to work. 
I appreciate the bug fix for the example, but I think the bigger fix might be to understand why ofGetWindowPosition is not working anymore.

Reply:
Hi, thanks for the consideration!

If the code used to work, I agree! I can definitely dive into ofGetWindowPosition in order to understand the issue at a deeper level. 

Is it ok to share the insights on this matter in this conversation? If you prefer, we close the PR and we open a new one when I find something interesting.

Reply:
In fact, the following code is already an issue. Even if it looks like the window is beeing set at exactly the same position it already has, in reality the window keeps moving. Precisely, each time it's set at +6 units in x and +27 units in y, with respect to the current position.  

```
void ofApp::update(){  
		int posx = ofGetWindowPositionX();
		int posy = ofGetWindowPositionY();		
		ofSetWindowPosition(posx, posy);
}
```

Reply:
Update: I found out the thing.  

![Screenshot 2024-12-08 213041](https://github.com/user-attachments/assets/fb435f4b-1069-4f34-88aa-4c55be05421e)

With reference to the figure above:  
ofSetWindowPosition sets the position of the red corner (point A).  
ofGetWindowPositionX/Y gets the position of the blue corner (point B).  
  
Therefore, the code I showed above retrieves the position of the internal frame and sets the external frame to that position. The net effect is that the window translates by (+6, +27)* each time.

*I verified that (6, 27) is the position of point B with respect to point A. 

I don't know if that's the intended behaviour of the functions, or if it arises due to the fact I'm working on WSL.

Reply:
> which platform are you in? it seems the dimensions of window decoration, status bar and border

Wow, you said it 2 weeks ago. Ops 😆  

Sorry, I just realized. Yes, it seems that's definitely the point.